### DISCUSSION TITLE
How to set tab title dynamically

### DISCUSSION THREAD

**Discussion by maku** [microsoft/terminal#16961] [Q&A]
I would like the title tab to be automatically set to the name of the current directory. Is this possible?. Of course, it would be best if this could also be customized (e.g. if the directory /x/y/z is active then display the title "foo")

**Comment by zadjii-msft** [comment]
This is certainly possible to do. Implementation would vary slightly based on the shell you're using. Are you using CMD, PowerShell, bash, something else?

All of these shells have different ways to modify what the prompt is, and you can usually update the title when the prompt is emitted.
